### PR TITLE
feat(*): update ormbfile struct

### PR DESCRIPTION
**Pull request by judgeeeeee**
**What this PR does / why we need it**:

fix irregular:
  Tensor.OpType json/yaml tags 
  Metadata.Hyperparameters
  Tensor.DType json/yaml tags
                   
update Signature.layers definition

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #149 

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
change: ormbfile Tensor.OpType json/yaml tag from  'optype'  to  'opType' 
change: ormbfile Tensor.DType json/yaml tag from  'dtype'  to  'dType' 
change: ormbfile 'Metadata.Hyperparameters' to 'Metadata.HyperParameters'
change: ormbfile 'Hyperparameter' to 'HyperParameter'
change: ormbfile Signature.Layers from '[]layer{name string}' to 'map[string]int'
change: model format from MXNETParams to MXNetParams
```

### CODE REVIEW

**Review by FogDong — COMMENTED** [review]
LGTM /cc @simon-cj

File: pkg/model/types.go [hunk 1]
```diff
@@ -45,23 +45,19 @@ type Hyperparameter struct {
...
 	Inputs  []Tensor `json:"inputs,omitempty" yaml:"inputs,omitempty"`
 	Outputs []Tensor `json:"outputs,omitempty" yaml:"outputs,omitempty"`
-	Layers  []Layer  `json:"layers,omitempty" yaml:"layers,omitempty"`
+	Layers  map[string]int  `json:"layers,omitempty" yaml:"layers,omitempty"`
 }
 
 type Tensor struct {
 	Name  string `json:"name,omitempty" yaml:"name,omitempty"`
 	Size  []int  `json:"size,omitempty" yaml:"size,omitempty"`
 	DType string `json:"dtype,omitempty" yaml:"dtype,omitempty"`
```
Comment (FogDong) [inline pkg/model/types.go:54]:
```suggestion
	DType string `json:"dType,omitempty" yaml:"dType,omitempty"`
```

**Comment by codeflitting** [comment]
LGTM

**Comment by judgeeeeee** [comment]
@simon-cj PTAL

**Pull request merged**